docassemble/assemblylinewizard/interview_generator.py

```diff
@@ -1148,3 +1153,231 @@ def is_person(field_name, people_vars=generator_constants.PEOPLE_VARS):
 
 def get_person_identifier(field_name):
   pass
+
+def create_package_zip(pkgname: str, info: dict, author_info: dict, folders_and_files: dict, fileobj:DAFile=None)->DAFile:
```

Comment:
Looks great. My only (optional) suggestion is that this function have it's own module, since it does have a separate function from the rest of the interview generator, and could be modified for people who want to fork, similar to `generator_constants.py`?

Reply:
This is a good idea for future.